content/ui-patterns/overlays.mdx

```diff
@@ -0,0 +1,415 @@
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 2fr']} gridGap={4}>
+    <div>
+        <Link href="https://primer.style/react/ActionMenu">
+            <img alt="Action menu" src="https://user-images.githubusercontent.com/293280/123880964-95b3a700-d8f8-11eb-9775-cbaf165207ed.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Action menu</Heading>
+        <Box as="p">
+
+Action menus are used for disambiguation, navigation, or to display secondary options. They appear when users interact with buttons, actions, or other controls.
+
+[Primer React implementation](https://primer.style/react/ActionMenu)
+
+</Box>
+    </div>
+    <div>
+        <Link href="https://primer.style/react/DropdownMenu">
+            <img alt="Dropdown menu" src="https://user-images.githubusercontent.com/293280/123881067-cdbaea00-d8f8-11eb-98e4-e57c64489308.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Dropdown menu</Heading>
+        <Box as="p">
+Dropdown menus are used for making a single selection among a small list of options. The list is usually predefined with system values, but in some cases it can include user-provided data when those are known not to grow into too many items.
+
+[Primer React implementation](https://primer.style/react/DropdownMenu)
+
+</Box>
+    </div>
+    <div>
+        <Link href="https://primer.style/react/SelectPanel">
+            <img alt="Select panel" src="https://user-images.githubusercontent.com/293280/123876997-4158f900-d8f1-11eb-85cb-461d9bbee0cb.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Select panel</Heading>
+        <Box as="p">
+
+Select panels allows manipulating long lists of options, with filtering and other advanced interactions. They can be used for single or multi-selection.
+
+[Primer React implementation](https://primer.style/react/SelectPanel)
+
+</Box>
+    </div>
+    <div>
+        <Link href="https://primer.style/react/Dialog2#confirmationdialog">
+            <img alt="Confirmation dialog" src="https://user-images.githubusercontent.com/293280/146620807-b5a757bc-59a0-4596-ad3a-ef656c7d5620.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Confirmation dialog</Heading>
+        <Box as="p">
+
+Confirmation dialogs lets users confirm or cancel an action. They are used as an extra friction for destructive actions.
+
+[Primer React implementation](https://primer.style/react/Dialog2#confirmationdialog)
+
+</Box>
+    </div>
+<div>
+        <Link href="https://primer.style/react/Dialog2">
+            <img alt="Dialog" src="https://user-images.githubusercontent.com/293280/146621670-20a7e5dd-1f62-4780-a7ed-10711db93fcb.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Dialog</Heading>
+        <Box as="p">
+
+Dialogs are larger used to display information, forms and other rich content.
```

Comment:
```suggestion
Dialogs are larger used to display information, forms and other rich content.
```

Feels like a few words are missing here.

Reply:
```suggestion
Dialogs are larger and used to display more information, forms, and other rich content.
```

Is this more along the lines of what is intended?

Reply:
that sounds good to me